MvvmCross/Core/Binding/Bindings/Target/Construction/MvxTargetBindingFactoryRegistry.cs

```diff
@@ -116,15 +116,16 @@ private IMvxPluginTargetBindingFactory FindSpecificFactory(Type type, string nam
             {
                 return factory;
             }
+            var baseType = type.GetTypeInfo().BaseType;
+            if (baseType != null)
+                factory = this.FindSpecificFactory(baseType, name);
+            if (factory != null) return factory;
```

Comment:
Will we ever reach the implementedInterfaces code?

It is only reached in the case that there is a factory. Are we interested in going there if we already have a factory?